packages/satusehat/src/Utils.js

```diff
@@ -3,7 +3,7 @@ import {
   request as commonRequest,
   logResponse,
 } from '@openfn/language-common/util';
-import pkg from '../package.json'  assert { type: 'json' };
+import pkg from '../package.json';
```

Comment:
I'm confused by this commit! I expect this to break the adaptor, and  certainly unit tests are failing right now